guides/common/attributes-orcharhino.adoc

```diff
@@ -46,6 +46,12 @@
 :smartproxy-example-com: orcharhino-proxy.example.com
 :smartproxy_port: 9090
 :fdi-package-name: orcharhino-fdi
+// Overwrite to avoid "Insights on Foreman"
```

Comment:
I wonder if instead of adding a separate set of orcharhino attributes we could instead move lines 51-54 to a shared attributes file and then only define `Insights` in the branded attribute files. Do you think that could work?